app/src/hooks/useLocalStorage.tsx

```diff
@@ -1,23 +1,23 @@
-import { DefaultState } from '@context/state'
+import { SessionData } from '@customTypes/common'
 import { useState } from 'react'
 
 // Custom hook to persist state across page refresh
 export default function useLocalStorage(
   key: string,
-  initialValue: DefaultState
+  initialValue: SessionData
 ) {
   // Pass initial state function to useState so logic is only executed once
-  const [storedValue, setStoredValue] = useState<DefaultState>(() => {
+  const [storedValue, setStoredValue] = useState<SessionData>(() => {
     if (typeof window === 'undefined') {
       return initialValue
     }
     try {
       // Get from local storage by key
-      const item: string | null = window.localStorage.getItem(key)
+      const item: string | null = window.sessionStorage.getItem(key)
```

Comment:
@karinamzalez There are some changes here that switch us from localStorage to sessionStorage for more temporary data persistence.